extension/src/popup/components/sendPayment/SendSettings/TxTimeout/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { Formik, Form, Field, FieldProps } from "formik";
+import { object as YupObject, number as YupNumber } from "yup";
+import { Input, Icon, Button } from "@stellar/design-system";
+import { useTranslation } from "react-i18next";
+
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import { FormRows } from "popup/basics/Forms";
+import { InfoTooltip } from "popup/basics/InfoTooltip";
+import { View } from "popup/basics/layout/View";
+import { SubviewHeader } from "popup/components/SubviewHeader";
+import {
+  saveTransactionTimeout,
+  transactionDataSelector,
+} from "popup/ducks/transactionSubmission";
+
+import "./styles.scss";
+
+export const SendSettingsTxTimeout = ({ previous }: { previous: ROUTES }) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { transactionTimeout } = useSelector(transactionDataSelector);
+
+  return (
+    <React.Fragment>
+      <SubviewHeader
+        title="Transaction Timeout"
+        customBackAction={() => navigateTo(previous)}
+        customBackIcon={<Icon.Close />}
+        rightButton={
+          <InfoTooltip
+            infoText={
+              <span>
+                {t(
+                  "Number of seconds that can pass before this transaction can no longer be accepted by the network",
+                )}{" "}
+              </span>
+            }
+            placement="bottom"
+          >
+            <></>
+          </InfoTooltip>
+        }
+      />
+
+      <Formik
+        initialValues={{ transactionTimeout }}
+        onSubmit={(values) => {
+          dispatch(saveTransactionTimeout(values.transactionTimeout));
+          navigateTo(previous);
+        }}
+        validationSchema={YupObject().shape({
+          transactionTimeout: YupNumber().min(
+            0,
+            `${t("must be greater than")} 0`,
+          ),
+        })}
+      >
+        {({ values, isValid, errors }) => (
+          <Form className="View__contentAndFooterWrapper">
+            <View.Content>
+              <FormRows>
+                <Field name="transactionTimeout">
+                  {({ field }: FieldProps) => (
+                    <Input
```

Comment:
I think we should offer a default value here. If I didn't know much about timeouts, I wouldn't know what to put here

Reply:
We do get the default from the tx store here so it defaults to the previous hardcoded value of 180